components/automl/create_model_for_tables/component.py

```diff
@@ -0,0 +1,44 @@
+from typing import NamedTuple
```

Comment:
suggestion to add copy right / License terms

Reply:
Done